Cabal/src/Distribution/Simple/GHC/ImplInfo.hs

```diff
@@ -88,6 +90,7 @@ ghcVersionImplInfo ver =
   GhcImplInfo
     { supportsHaskell2010 = v >= [7]
     , supportsGHC2021 = v >= [9, 1]
+    , supportsGHC2024 = v >= [9, 9]
```

Comment:
`9, 9` or `9, 10` here?

Reply:
9.9 refers to an unreleased GHC built from HEAD after the 9.8 fork and before 9.10. So this only really affects developers, and we don't actually know whether `GHC2024` is supported or not, but I presume it should be allowed to try. Similarly `supportsGHC2021` is defined to allow 9.1 and later, since `GHC2021` was first released with GHC 9.2.